src/VisualStudio/Xaml/Impl/Features/Commands/IXamlCommandService.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.VisualStudio.LanguageServices.Xaml.Features.Commands
+{
+    internal interface IXamlCommandService : ILanguageService
+    {
+        /// <summary>
+        /// Execute the <paramref name="command"/> with the <paramref name="commandArguments"/> on server
```

Comment:
>on server [](start = 96, length = 9)

That's just an implementation detail that's not relevant to the description of the interface

Reply:
Sure I will update the comment.